repository/p.json

```diff
@@ -2261,6 +2261,17 @@
 				}
 			]
 		},
+		{
+			"name": "PrintDebugger",
+			"details": "https://github.com/futureprogrammer360/PrintDebugger",
+			"labels": ["debug", "print", "output"],
+			"releases": [
+				{
+					"sublime_text": "*",
```

Comment:
since you use `py3.8`, this should be at least be `"sublime_text": ">=4050"` (first dev build) or even `"sublime_text": ">=4107"` (first stable build) which also includes the new `chain` command from Build 4103.